utils/file-manager.js

```diff
@@ -94,9 +94,12 @@ export default class FileManager {
    */
   async init(uppyFiles) {
     this.fileList = await FileManager.extractZips(uppyFiles)
-    this.fileDict = FileManager.removeTopmostFilenames(
-      this.fileList.map(f => [f.name, f])
-    )
+    const filePairs = this.fileList.map(f => [f.name, f])
+    if (filePairs.length === 1) {
+      this.fileDict = FileManager.removeTopmostFilenames(filePairs)
```

Comment:
Maybe we should change how `removeTopmostFilenames` behaves instead, not sure what makes the most sense